### PR TITLE
Add advanced

### PR DESCRIPTION
**Pull request by christopher-hakkaart**
- Add advanced training to nextflow-io

### CODE REVIEW

**Review by mribeirodantas — APPROVED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -34,6 +34,16 @@ Find the one that's right for you:
...
 
+!!! exercise "Advanced training"
+
+    !!! quote inline end ""
+
+        :material-lightbulb: This is the advanced Nextflow training material used in advanced training events.
+
+    An advanced material explores the advanced features of the Nextflow language and runtime, and how to use them to write efficient and scalable data-intensive workflows.
+
+    [Launch the Hands on Training :material-arrow-right:](advanced/index.md){ .md-button .md-button--primary }
```
Comment (mribeirodantas) [inline docs/index.md:45]:
```suggestion
    [Launch the Advanced Training :material-arrow-right:](advanced/index.md){ .md-button .md-button--primary }
```

**Review by mribeirodantas — APPROVED** [review]

**Comment by mribeirodantas** [comment]
LGTM?

**Comment by robsyme** [comment]
This is so great guys, can't thank you enough. I'd like to add a couple more modules focused on `publishDir` and configuration, but I'm happy to add those in another PR if you'd like to get your work here in.

**Comment by christopher-hakkaart** [comment]
@robsyme you're welcome to work on this or or make a new pr. It's pre-approved already so it should be g2g either way

**Comment by ewels** [comment]
Hi folks! Are we ready to merge here?

**Comment by robsyme** [comment]
Two last modules to add before merge: 
- publishDir and
- configuration modules
Which I'll push here before the end of the week.

File: docs/index.md [hunk 2]
```diff
@@ -34,6 +34,16 @@ Find the one that's right for you:
 
     [Start the Nextflow Training Workshop :material-arrow-right:](basic_training/index.md){ .md-button .md-button--primary }
 
+!!! exercise "Advanced training"
+
+    !!! quote inline end ""
+
+        :material-lightbulb: This is the advanced Nextflow training material used in advanced training events.
```
Comment (mribeirodantas) [inline docs/index.md:41]:
```suggestion
        :material-lightbulb: This is the Nextflow training material used in advanced training events.
```

File: docs/index.md [hunk 3]
```diff
@@ -34,6 +34,16 @@ Find the one that's right for you:
...
+
+    An advanced material explores the advanced features of the Nextflow language and runtime, and how to use them to write efficient and scalable data-intensive workflows.
```
Comment (mribeirodantas) [inline docs/index.md:43]:
```suggestion
    An advanced material exploring the advanced features of the Nextflow language and runtime, and how to use them to write efficient and scalable data-intensive workflows.
```

**Pull request merged**